packages/commonwealth/client/scripts/views/pages/Community/Members/CommunityMembersPage.tsx

```diff
@@ -0,0 +1,218 @@
+
+  const totalResults = members?.pages?.[0]?.totalResults || 0;
+
+  // fixes bug that prevents scrolling on initial page load
+  useEffect(() => {
+    const shouldFetchMore = formattedMembers.length < 50 && totalResults > 50;
+    if (!shouldFetchMore) {
+      return;
+    }
+    fetchNextPage();
+  }, [formattedMembers, totalResults, fetchNextPage]);
+
+  useEffect(() => {
+    // Set the active tab based on URL
+    if (
+      window.location.search.includes(
+        `tab=${TABS[0].split(' ').join('-').toLowerCase()}`
+      )
+    ) {
+      setSelectedTab(TABS[0]);
+    } else if (
+      isGatingEnabled &&
+      window.location.search.includes(
+        `tab=${TABS[1].split(' ').join('-').toLowerCase()}`
+      )
+    ) {
+      setSelectedTab(TABS[1]);
+    } else {
+      setSelectedTab(TABS[0]);
+    }
+  }, []);
+
+  const navigateToCreateGroupPage = () => {
+    navigate(`/${app.activeChainId()}/members/groups/create`);
+  };
+
+  const isAdmin = Permissions.isCommunityAdmin() || Permissions.isSiteAdmin();
+
+  return (
+    <section className="CommunityMembersPage">
+      {/* TODO: add breadcrums here */}
+
+      {/* Header */}
+      <CWText type="h2">Members ({totalResults})</CWText>
+
+      {/* Tabs section */}
+      <CWTabBar>
+        {TABS.map((tab, index) => (
+          <CWTab
+            key={index}
+            label={tab}
+            onClick={() => setSelectedTab(tab)}
+            isSelected={selectedTab.toLowerCase() === tab.toLowerCase()}
+          />
+        ))}
+      </CWTabBar>
+
+      {/* Filter section */}
+      <section
+        className={getClasses<{
+          'cols-3': boolean;
+          'cols-4': boolean;
+        }>(
+          {
+            'cols-3': !isAdmin,
+            'cols-4': isAdmin,
+          },
+          'filters'
+        )}
+      >
+        <CWTextInput
+          size="large"
+          fullWidth
+          placeholder={`Search ${
+            selectedTab === TABS[0] ? 'members' : 'groups'
+          }`}
+          iconLeft={<CWIcon iconName="search" className="search-icon" />}
+          onInput={(e) =>
+            setSearchFilters((g) => ({
+              ...g,
+              searchText: e.target.value?.trim(),
+            }))
+          }
+        />
+        {isGatingEnabled && (
+          <div className="select-dropdown-container">
+            <CWText type="b2" fontWeight="bold" className="filter-text">
+              Filter
+            </CWText>
+            <Select
+              containerClassname="select-dropdown"
+              options={GROUP_FILTERS.map((x) => ({
+                id: x,
+                label: x,
+                value: x,
+              }))}
+              selected={searchFilters.category}
+              dropdownPosition="bottom-end"
+              onSelect={(item: any) => {
+                setSearchFilters((g) => ({ ...g, category: item.value }));
+              }}
+            />
+          </div>
+        )}
+        {isAdmin && (
```

Comment:
I think this button also should be behind the feature flag for now. Now it takes me to 404

Reply:
this is actually related to the gating forms. This will not give a 404 after https://github.com/hicommonwealth/commonwealth/pull/5045 is merged